CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ All versions prior to 0.0.9 are untracked.
 
 ## [Unreleased]
 
+## [2.4.10]
+
+### Fixed
+
+* Fixed a crash triggered when no vulnerabilities are found with some
```

Comment:
My mistake, I didn't add a CHANGELOG entry when I merged this in. I didn't realise this had gone out into a release until now as the user who made this PR was testing trunk.

Reply:
Sorry for being unclear. I was testing *my* trunk with released `pip-audit`, not your trunk!

Reply:
No worries! I should have known a release was cut anyway.